scripts/skill_registry.py

```
@@ -0,0 +1,436 @@
+
+        if args.format == "json":
+            print(format_json(skills))
+        else:
+            print(format_markdown(skills))
+
+        return 0
+
+    except KeyboardInterrupt:
+        print("\nInterrupted", file=sys.stderr)
+        return 1
+    except Exception as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)